### ISSUE TITLE
convert.py throwing error

### ISSUE THREAD

**Issue by bryanapple** [MothSynths/MothSynthPy#2]
Using the samples and binaries from the this git repo...

File "MothSynthPy-main/convert.py", line 219, in <module>
    hashBytes = bytes.fromhex(hash) #convert hash to bytes
                ^^^^^^^^^^^^^^^^^^^
ValueError: non-hexadecimal number found in fromhex() arg at position 1

**Comment by bryanapple** [comment]
The hash seems to hold unexpected values:
  Detected image type: ESP8266
  WARNING: Suspicious segment 0xee,

**Comment by MothSynths** [comment]
We only support esp32, maybe that's the issue?

**Comment by bryanapple** [comment]
Agree.  This was run with the files (.bin) from this repo.  Are these files correct?
